spec/features/support/env.rb

```diff
@@ -0,0 +1,21 @@
+# Generated by cucumber-sinatra. (2015-01-16 11:21:49 +0000)
```

Comment:
this is a cucumber-sinatra init generated file - remove it as you are not using cucumber